Gateway/Http/Client/RefundClient.php

```diff
@@ -68,8 +70,16 @@ public function placeRequest(TransferInterface $transferObject): array
             $payloadData = $transferObject->getBody();
             $isFullRefund = $this->isFullRefund($payloadData);
             $refund = $this->refund($payloadData, $isFullRefund);
+        } catch (AlmaClientException $e) {
+            $fails = new stdClass();
+            $fails->responseCode = 460;
+            $fails->message = $e->getMessage();
+            return [
+                'resultCode' => 0,
+                'fails' => $fails,
+            ];
         } catch (RequestError $e) {
-            $this->logger->error("Error creating refund:", [$e]);
+            $this->logger->error("Error creating refund:", [$e->getMessage()]);
```

Comment:
warning, sometime, getMessage is empty in `RequestError`

with [php-client 1.9.0](https://github.com/alma/alma-php-client/releases/tag/v1.9.0), you can now use `getErrorMessage`